.buildkite/install-lmcache.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+CONDA_ENV_NAME="buildkite"
+PYTHON_VERSION=3.10
+
+cuda_version=12.1
+export CUDA_HOME=/usr/local/cuda-${cuda_version}
+export LD_LIBRARY_PATH=$CUDA_HOME/lib64:$LD_LIBRARY_PATH
```

Comment:
My gut feeling is that there are too many bash scripts. We are definitely going to add more scripts in the future, so it would be ideal if we can reduce the # of bash scripts.

Reply:
Merged install-env.sh and install-packages.sh